test/environmentSettings.ts

```diff
@@ -10,6 +10,26 @@ import { join } from 'path';
 export class EnvironmentSettings {
   private static _instance: EnvironmentSettings;
 
+  private _vscodeVersion = 'stable'; //  or 'insiders' or '1.77.3'
```

Comment:
Do you think we should update our e2e workflow to take this into consideration for automation? Or leave it just in here for now?